docs/configuration.md

```diff
@@ -1342,6 +1342,19 @@ Apart from these, the following properties are also available, and may be useful
     Should be greater than or equal to 1. Number of allowed retries = this value - 1.
   </td>
 </tr>
+<tr>
+  <td><code>spark.scheduler.taskAssigner</code></td>
+  <td>roundrobin</td>
+  <td>
+    The strategy of how to allocate tasks among workers with free cores. By default, roundrobin
```

Comment:
nit: create a list for each policy and explain inline instead of saying `former`, `latter` below.